src/main/java/wolox/training/config/SwaggerConfig.java

```diff
@@ -0,0 +1,39 @@
+package wolox.training.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+
+    @Bean
+    public Docket productApi() {
```

Comment:
Just a comment here: Why is the name of this method `productApi`? 🤔 
It's usually called just `api`

Reply:
Good catch, I've just changed it! I made that mistake by copypasting from another site that piece of code :sweat_smile: